app/src/main/res/values/strings.xml

```diff
@@ -271,4 +271,5 @@
     <string name="starred">Starred</string>
     <string name="set_scale">Set scale</string>
     <string name="include_if_asking_support">If you want to request support, include the following info:</string>
+    <string name="duplicateCard">Duplicate Card</string>
```

Comment:
Given we're looking at a card and the other options are called things like "Share" instead of "Share Card", I think this makes more sense without "Card".